### PR TITLE
Add postgres in docker

### PR DESCRIPTION
**Pull request by hexponent**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,11 +3,23 @@ services:
...
+      - POSTGRES_PASSWORD=123456
+      - POSTGRES_DB=postgres
+    ports:
+      - "5432:5432"
   bot:
     hostname: bot
     image: flea
     environment:
       - REDIS_URL=redis
+      - DATABASE_URL=postgres://postgres:123456@db:5432/postgres
```
Comment (mykolasolodukha) [inline docker-compose.yml]: Is it safe to run a `postgres` instance with such open credentials, given that we expose the `5432` port to external connections (line 16)?

**Review by mykolasolodukha — CHANGES_REQUESTED** [review]
Safety concerns

File: docker-compose.yml (hunk 1 above)
Comment (hexponent) [reply docker-compose.yml]: You are right, its better to hide it, but I was waiting for postgres code to be merged. I will hide it here and also variables for db container as well

**Comment by Salz0** [comment]
@hexponent hey, if you could solve the conflicts and do the safety measure -- would be awesome! Will merge.

Postgres is now activated within the bot

**Review by Salz0 — CHANGES_REQUESTED** [review]
safety concern

File: docker-compose.yml (hunk 1 above)
Comment (hexponent) [reply docker-compose.yml]: @mykolasolodukha fixed

**Comment by Salz0** [comment]
Thank you! @hexponent

**Pull request merged**